### DISCUSSION TITLE
Need help to update an automation

### DISCUSSION THREAD

**Discussion by berma83** [Budibase/budibase#3068] [Help]
Hi!

I'm trying to modify a field in automation but I can't find a save button.

I need to update the field '''prodaux1''' but it always back to previous value.

What am I doing wrong?

See below:

https://user-images.githubusercontent.com/53205779/137758114-c08f80a2-99bc-4094-beda-3bf676cc3699.mp4

**Comment by PClmnt** [comment]
Hey @berma83 I'm going to raise an issue for this and merge a fix in
